v0/sources/shopify/data/identifyMapping.json

```diff
@@ -32,6 +32,11 @@
         "sourceKeys":"default_address",
         "destKeys":"traits.address"
     },
+
+    {
+        "sourceKeys":"shipping_address",
```

Comment:
Along the same lines, wouldn't `billing_address` also make its way here?

Reply:
Fixed